### ISSUE TITLE
End of encounter loot

### ISSUE THREAD

**Issue by GeorgeLung** [theGenebob/TowerofAdrenspire#1]
Firstly got here via your reddit post and I think this is great.

Completed the first encounter and when I tried to use the loot chest it didn't do what I expected. My intention was to transfer all items (2 gold pieces) to my character. So I selected him and clicked loot. Instead it transferred all the items from my character to the chest.

**Comment by theGenebob** [comment]
Yes, I am using the built in "loot actor" from pf2e and can see how that would not be intuitive. I'm going to be begin working on an alternative now.
